docs/user-guide-for-quickstart.md

```diff
@@ -0,0 +1,133 @@
+In the pipeline page, we could see there are 5 pipelines, click 'START ALL PIPELINES' button, we could start all the predefined pipelines.
+
+<div class="my-carousel docs-carousel">
+    <img src="/img/quickstart/user-guide/02_pipelines.png" alt="Pipeline Overview"/>
+</div>
+
+
+ Using the 'Water level trend analysis' pipeline as an example. By clicking the 'Show pipeline' button, we could see the details of the pipeline as the following figure.
+ 
+
+The pipeline contains 3 major parts:
+- **Data source**: 
+  - Machine Data Simulator (Water Level).
+- **Data Processors**: 
+  - Trend Detector: to analyse the water level increase speed within specified time, when Trend Detector find the speed is reaching the limit, it will trigger the notification. 
+  - Welford Change Detection: to calculate the variance of the mean and change of the water level over a specified time 
+  - Boolean Counter: to count each 'overflow' status change from 'false' to 'true'. 
+  - Numerical Filter: when 'overflow' status change from 'false' to 'true' reaches 5 times, it will trigger Notification.
+- **Data Sinks**: 
+  - Notification: to push a notification when the measured parameter condition reaches the set condition of the trigger, which are sent to Notification interface. 
+  - Data Lake: to store the events in the internal data lake, offering data for dashboard and data explorer.
+
+<div class="my-carousel docs-carousel"> 
+<img src="/img/quickstart/user-guide/07_water_level_trend_pipeline.png" alt="Pipeline Detail"/>
+</div>
+
+## Connect
+The _Connect_ view provides an overview of all existing adapters to let StreamPipes connect data sources.
+
+With StreamPipes Connect it is possible to connect new data sources in StreamPipes with just a few clicks.
+It is also possible to connect specific data sources or connect generic sources like message brokers or databases.
+If the event schema of the data source is unknown, the system tries to infer the schema by extracting some sample data and analysing it.
+
+In the Quickstart Mode, we use the Machine Data Simulator and Data stream to simulate the factory machine condition data and environmental data, just click 'START ALL ADAPTERS' button, we could start all existing adapters to offer data source for pipelines.
+
+
+<div class="my-carousel docs-carousel">
+    <img src="/img/quickstart/user-guide/03_adapters.png" alt="Adapters"/>
+</div>
+
+
+
+## Dashboard
+The _Dashboard_ can be used for multiple use cases.
+
+It is a good way to live monitor a running system in a dashboard, but it can also be used during the pipeline development to get a fast feedback from newly created pipelines.
+Currently 10 different types are available, including line charts, various map visualizations and data tables. The Dashboard feature allows you to select and view real-time data from specific pipelines that interest you.
```

Comment:
```suggestion
Currently 10 different types of visualizations are available, including line charts, various map visualizations and data tables. The Dashboard feature allows you to select and view real-time data from specific pipelines that interest you.
```

Reply:
Hi, @bossenti ! I have updated the description in this part.